cli/tests/unit/process_test.ts

```diff
@@ -30,6 +31,46 @@ unitTest({ perms: { run: true } }, async function runSuccess(): Promise<void> {
   p.stdout.close();
   p.close();
 });
+unitTest({ perms: { run: true } }, async function runStdinRid0(): Promise<
+  void
+> {
+  const p = Deno.run({
+    cmd: ["python", "-c", "print('hello world')"],
+    stdin: 0,
```

Comment:
I'm confused. Certainly Deno shouldn't panic when given 0 for stdin, but this is showing it be successful. Shouldn't this throw? What does `stdin: 0` mean?

Reply:
It means use `rid 0` as `stdin`. And `rid 0` maps to the current process stdin:

```
> Deno.resources()
{ 0: "stdin", 1: "stdout", 2: "stderr", 3: "repl" }
```

Reply:
It means use `rid 0` as `stdin`. And `rid 0` maps to the current process stdin:

```
> Deno.resources()
{ 0: "stdin", 1: "stdout", 2: "stderr", 3: "repl" }
```

Reply:
Ah cool. I forgot we had this feature.